src/modules/Game.class.js

```diff
@@ -23,22 +23,156 @@ class Game {
   constructor(initialState) {
     // eslint-disable-next-line no-console
     console.log(initialState);
+
+    // Ініціалізуємо гральну дошку
+    this.defaultState = [
+      [0, 0, 0, 0],
+      [0, 0, 0, 0],
+      [0, 0, 0, 0],
+      [0, 0, 0, 0],
+    ];
+
+    this.board = initialState || JSON.parse(JSON.stringify(this.defaultState));
+    this.score = 0; // Початкови рахунок
+    this.status = 'idle'; // Початковий статус гри
+  }
+
+  moveLeft() {
+    this.checkLose();
```

Comment:
it should be done in after move - same as checkWin (no need to do it in each "move" method)